templates/navbar.php

```diff
@@ -30,8 +30,8 @@
     <div class="navbar-collapse collapse">
       <ul class="nav navbar-nav">
         <!--<li><a href="index.php">Home</a></li>-->
-        <li id='ipdatabase.php?fav=0' class=''><a href="ipdatabase.php?fav=0"><i class='fa fa-laptop fa-lg' aria-hidden='true'></i>&nbsp; IPs</a></li>
-        <li id='ipdatabase.php?fav=1' class=''><a href="ipdatabase.php?fav=1"><i class='fa fa-star fa-lg' aria-hidden='true'></i>&nbsp; Favoriten</a></li>
+        <li id='navIPs' class=''><a href="ipdatabase.php?fav=0"><i class='fa fa-laptop fa-lg' aria-hidden='true'></i>&nbsp; IPs</a></li>
+        <li id='navFavouriteIPs' class=''><a href="ipdatabase.php?fav=1"><i class='fa fa-star fa-lg' aria-hidden='true'></i>&nbsp; Favoriten</a></li>
```

Comment:
Nope nicht wirklich. Ich hab die nur übernommen wie sie waren 😂 copy and paste halt haha. Wollte schon fragen ob das nen Grund hat?